quaint/src/connector.rs

```diff
@@ -10,35 +10,80 @@
 //! querying interface.
 
 mod connection_info;
+
+#[cfg(not(target_arch = "wasm32"))]
 pub(crate) mod metrics;
```

Comment:
Internal: as planned, `metrics` will not make it to the Wasm Query Engine as they are.